sklearn_numba_dpex/kmeans/kernels.py

```diff
@@ -71,16 +72,16 @@ def make_fused_fixed_window_kernel(
     # (NB: it's already supported by numba.cuda) X_t should be an input to the factory
     # rather than an input to the kernel.
     @dpex.kernel
-    # fmt: off
     def fused_kmeans(
+        # fmt: off
         X_t,                               # IN READ-ONLY   (n_features, n_samples)
         current_centroids_t,               # IN             (n_features, n_clusters)
         centroids_half_l2_norm,            # IN             (n_clusters,)
         per_sample_pseudo_inertia,         # OUT            (n_samples,)
-        new_centroids_t_private_copies,    # OUT            (n_private_copies, n_features, n_clusters)
-        cluster_sizes_private_copies,      # OUT            (n_private_copies, n_clusters)
+        new_centroids_t_private_copies,    # OUT            (n_private_copies, n_features, n_clusters)  # noqa
+        cluster_sizes_private_copies,  # OUT            (n_private_copies, n_clusters)  # noqa
     ):
-    # fmt: on
+        # fmt: on
```

Comment:
Question: what does this stand for?

Reply:
it was supposed to disable black within the block delimited by those tags